src/common/snippets/src/utils/tokenization_utils.cpp

```diff
@@ -387,13 +388,13 @@ bool tokenize_node(const std::shared_ptr<ov::Node>& node, const SnippetsTokeniza
     }
 
     // The each data node (Parameter (and non-Scalar Constants), Result, Buffers with the same ID) requires the own
-    // unique GPR. At the moment, CPU Plugin has limitation for GPR registers: there are 12 available GPRs, and one of
-    // them must be reserved for runtime parameters, so only 11 can be used during kernel execution. This limitation
-    // will be resolved once generator supports gprs spills [75622].
+    // unique GPR. At the moment, CPU Plugin has limitation for GPR registers.
+    // This limitation will be resolved once generator supports gprs spills [75622].
     // TODO [75567]: move this plugin-specific constraint to the plugin callback
+    const auto io_count = body_parameters.size() + body_results.size() + hidden_data_count;
     const auto unique_buffer_count = op::Subgraph::get_estimated_buffer_count(ops_for_buffer_count);
-    const size_t max_data_ptr_count = config.get_data_ptr_gpr_count();
-    if (body_parameters.size() + body_results.size() + hidden_data_count + unique_buffer_count > max_data_ptr_count) {
+    static constexpr size_t loops_depth = 2;
```

Comment:
why here we have 2 (another place has 3 and there was a comment)? Shall we add a comment?